source/ways-of-working/new-component/gitops-flux.html.md.erb

```diff
@@ -17,30 +17,54 @@ One of our principles being <b>Everything as Code</b>, we practise [Gitops](http
 
 [Flux](https://docs.fluxcd.io/) is a tool that automatically ensures that the state of a cluster matches the config in [Git](https://github.com/hmcts/cnp-flux-config). 
 
-- It is an operator in the cluster to trigger deployments inside Kubernetes, which means you don't need a separate CD tool.
-- It monitors all relevant image repositories, detects new images, triggers deployments and updates the desired running configuration. 
+- Flux uses Gitops Toolkit, a set of APIs and controllers that make up the runtime for Flux.
```

Comment:
can we archive this page with a disclaimer at the top? SDS hasn't started migration and we share docs.